### ISSUE TITLE
Remove Highlight.js dependency install

### ISSUE THREAD

**Issue by endigo9740** [skeletonlabs/create-skeleton-app#42]
![Screenshot 2023-09-08 at 11 00 29 AM](https://github.com/skeletonlabs/create-skeleton-app/assets/1509726/de4476cb-dcf5-4c2a-be1e-30a1e39fcf3e)

While handy, we've identified that Highlight.js has updated over the last couple months and seems to be greatly inflating the final bundle size. This was brought up in this issue:

https://github.com/skeletonlabs/skeleton/issues/1381

We've also had a number of users run into this during the v2 RC testing phase. My guess is it's installing EVERY language module, instead of only those you need or want.

Given this, I'm going to recommend we temporarily remove the Highlight.js dependency to make this a manual install if folks wish to use it. A number of folks just opt to install everything without thinking or consideration for whether they will actually use it or not. Which means a lot of folks are bloating their app bundle for a feature they never use.

Once we can determine better instructions for limiting the size of Highlight, this can return.

**Comment by moritzhertler** [comment]
I might add that the current template only works in dev mode (see [#2024](https://github.com/skeletonlabs/skeleton/issues/2024)). All classes imported by 

```typescript 
import 'highlight.js/styles/github-dark.css';
```

will be removed by PurgeCSS during the build. The correct PurgeCSS config should be considered when this feature eventually returns.

I've also just run into this issue. After adding a code block with syntax highlighting my bundle size increased by ~900kB. I fixed it by only importing the languages I'm actually using (see [highlight.js usage](https://highlightjs.org/#usage)). I also tried dynamically importing languages which works nicely.
